src/crime_prediction/preprocessingCrimeData.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Mar 29 16:09:02 2018
+
+@author: valentinarodriguez
+"""
+
+
+
+def dataCleaning(data, columnsToDrop):
+    data = data.drop(columnsToDrop, axis=1)
+    data = data.fillna(0)
+    print("dataCleaning")
+    print(data)
+    return data
+
+
+def convertingCrimeCategoryNumericalToClass(data):
+     #violent crimes from 1 to 4
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=1,value= 1)
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=2,value= 1)
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=3,value= 1)
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=4,value= 1)
+      #violent crimes from 5 to 7
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=5,value=-1)
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=6,value=-1)
+     data["Statute_CrimeCategory"] = data["Statute_CrimeCategory"].replace(to_replace=7,value=-1)
+     return data
+
+
+def convertingCategorizedDataToNumerical(data):
```

Comment:
In each of the functions except _convertingCrimeCategoryNumericalToClass_ , we probably want to be using one-hot encoding. Is there an ordering I'm missing?